docs/en/Running-Flink-on-Alluxio.md

```diff
@@ -91,10 +105,18 @@ This example assumes you have set up Alluxio and Flink as previously described.
 
 Put the file `LICENSE` into Alluxio, assuming you are in the top level Alluxio project directory:
 
-{% include Running-Flink-on-Alluxio/license.md %}
+```bash
+$ bin/alluxio fs copyFromLocal LICENSE alluxio://localhost:19998/LICENSE
+```
 
 Run the following command from the top level Flink project directory:
 
-{% include Running-Flink-on-Alluxio/wordcount.md %}
+```bash
+$ bin/flink run examples/batch/WordCount.jar --input alluxio://localhost:19998/LICENSE --output alluxio://localhost:19998/output
+```
 
 Open your browser and check [http://localhost:19999/browse](http://localhost:19999/browse). There should be an output file `output` which contains the word counts of the file `LICENSE`.
+
+> Tips：The previous wordcount example is also applicable to Alluxio in fault tolerant mode with Zookeeper. 
```

Comment:
link to detailed docs so that users can decide for themselves whether to use config vs uri